packages/SwingSet/src/kernel/kernel.js

```diff
@@ -725,6 +726,7 @@ export default function buildKernel(kernelEndowments) {
 
     kernelKeeper.setInitialized();
     commitCrank(); // commit "crank 0"
+    kernelKeeper.incrementCrankNumber();
```

Comment:
ditto

Reply:
At this point it may be easier to see why the commit comes first.  That commit commits crank 0.  If we incremented first, it would be committed as crank 1.